### PR TITLE
Cleaned up error messages

### PR DESCRIPTION
**Pull request by ecumene**
This diff is a bit hard to follow, but I changed the error messages from this:
```
$ rustup update stable
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: downloading component 'rustc'
info: downloading component 'rust-std'
info: downloading component 'cargo'
info: downloading component 'rust-std' for 'armv7-apple-ios'
info: downloading component 'rust-std' for 'i686-apple-darwin'
error: component download failed for rust-std-i686-apple-darwin
info: caused by: could not download file from 'https://static.rust-lang.org/dist/2017-03-11/rust-std-1.16.0-i686-apple-darwin.tar.gz' to 'C:\Users\brian\.rustup\tmp\4y0bss25meytiiyq_file
info: caused by: error during download
info: caused by: [18] Transferred a partial file (transfer closed with 3544136 bytes remaining to read)
```

To this:

```
$ rustup update stable
info: syncing channel updates for 'stable-x86_64-pc-windows-msvc'
info: downloading component 'rustc'
info: downloading component 'rust-std'
info: downloading component 'cargo'
info: downloading component 'rust-std' for 'armv7-apple-ios'
info: downloading component 'rust-std' for 'i686-apple-darwin'
error: component download failed for rust-std-i686-apple-darwin
error: caused by: could not download file from 'https://static.rust-lang.org/dist/2017-03-11/rust-std-1.16.0-i686-apple-darwin.tar.gz' to 'C:\Users\brian\.rustup\tmp\4y0bss25meytiiyq_file
error: caused by: error during download
error: caused by: [18] Transferred a partial file (transfer closed with 3544136 bytes remaining to read)
```

As described in #991

### CODE REVIEW

**Comment by kinnison** [comment]
I don't see the change in messages in your posting.  Could you clarify what I'm supposed to be looking for?

Also, do not post formatting commits, instead reformat your existing commit please.  I can show you how to rebase if you need help with that.

**Comment by ecumene** [comment]
Hi! Thanks for the feedback! This is the first time I contributed on a project this size. Commit is squashed!

As for where you can see the errors, just turn off wifi and run `rustup default nightly`
```
info: syncing channel updates for 'nightly-x86_64-apple-darwin'
error: could not download file from 'https://static.rust-lang.org/dist/channel-rust-nightly.toml.sha256' to '/Users/mitchellhynes/.rustup/tmp/lupmzvmrx5e2nuha_file'
error: caused by: failed to make network request
error: caused by: https://static.rust-lang.org/dist/channel-rust-nightly.toml.sha256: error trying to connect: failed to lookup address information: nodename nor servname provided, or not known
error: caused by: failed to lookup address information: nodename nor servname provided, or not known
```

**Comment by kinnison** [comment]
I'm still failing to spot how the messages differ.  What is your intent?  (I could be being entirely blind here)

**Comment by ecumene** [comment]
The level for logging error details was `info`, when it should be `error`, which makes it easier to spot where the error is coming from and what's associated with the source of the error

**Comment by kinnison** [comment]
Oh my goodness, so it is.  I am so sorry, I was trying to work out how the colons were different the entire time :man_facepalming:

**Review by kinnison — APPROVED** [review]
This looks good to me.
